pkg/verifier/notation/notation.go

```diff
@@ -44,7 +44,7 @@ import (
 )
 
 const (
-	verifierName    = "notation"
+	verifierName    = "verifier-notation"
```

Comment:
Will changing the name of the notation verifier breaking existing versions (including current v1.0.0)?